### PR TITLE
asyncio.get_event_loop() is deprecated in 3.10, stop using it (mostly)

### PR DESCRIPTION
**Pull request by altendky**
https://docs.python.org/3.10/library/asyncio-eventloop.html#asyncio.get_event_loop
> Deprecated since version 3.10: Deprecation warning is emitted if there is no running event loop. In future Python releases, this function will be an alias of get_running_loop().

Notes:
- Usages in `event_loop()` fixtures are being explored for removal in https://github.com/Chia-Network/chia-blockchain/pull/10420.

### CODE REVIEW

**Comment by mariano54** [comment]
Reopening the PR here to re run tests

File: chia/full_node/full_node.py [hunk 1]
```diff
@@ -972,10 +972,7 @@ async def validate_block_batches(batch_queue):
                 await self.send_peak_to_wallets()
                 self.blockchain.clean_block_record(end_height - self.constants.BLOCKS_CACHE_SIZE)
 
-        loop = asyncio.get_event_loop()
-        batch_queue: asyncio.Queue[Tuple[ws.WSChiaConnection, List[FullBlock]]] = asyncio.Queue(
-            loop=loop, maxsize=buffer_size
-        )
+        batch_queue: asyncio.Queue[Tuple[ws.WSChiaConnection, List[FullBlock]]] = asyncio.Queue(maxsize=buffer_size)
```
[the remark this answers is not in the document]
Comment (altendky) [reply chia/full_node/full_node.py:975]: The loop parameter [was deprecated in 3.8](https://docs.python.org/3.9/library/asyncio-queue.html#asyncio.Queue) and [has been removed in 3.10](https://docs.python.org/3.10/library/asyncio-queue.html#asyncio.Queue).

File: chia/seeder/dns_server.py [hunk 2]
```diff
@@ -267,21 +283,7 @@ def main():
...
-        loop.add_signal_handler(signal.SIGINT, signal_received)
-        loop.add_signal_handler(signal.SIGTERM, signal_received)
-    except NotImplementedError:
-        log.info("signal handlers unsupported")
-
-    try:
-        loop.run_until_complete(serve_dns(config, root_path))
-    finally:
-        loop.close()
+    asyncio.run(async_main(config=config, root_path=root_path))
```
[the remark this answers is not in the document]
Comment (altendky) [reply chia/seeder/dns_server.py:286]: `chia start seeder` fails with `PermissionError: [Errno 13] Permission denied` opening the socket like `main` does anyways.

**Comment by jack60612** [comment]
The majority of this pr looks fine, however I am not sure what will happen with some things after the loop removal so I will let mariano approve it.

**Review by mariano54 — DISMISSED** [review]

**Review by wjblanke — APPROVED** [review]
aok

**Pull request merged**